src/java/com/zimbra/clam/ClamScannerConfig.java

```diff
@@ -18,7 +18,6 @@
 package com.zimbra.clam;
```

Comment:
changes in this file don't seem to be required

Reply:
Have removed unused imports.